src/clr-angular/emphasis/alert/alerts.spec.ts

```diff
@@ -68,6 +68,18 @@ describe("ClrAlerts component", function() {
             expect(testComponent.currentAlertIndex).toEqual(1);
         });
 
+        it("allows changing alert index from 1 to 0", function() {
+            this.create(TestComponent);
+            const testComponent = fixture.componentInstance;
+            fixture.detectChanges();
+            testComponent.currentAlertIndex = 1;
+            fixture.detectChanges();
+            expect(testComponent.alertsInstance.currentAlertIndex).toEqual(1);
+            testComponent.currentAlertIndex = 0;
+            fixture.detectChanges();
+            expect(testComponent.alertsInstance.currentAlertIndex).toEqual(0);
+        });
+
```

Comment:
This is great. But could you also check against `null` and `-1`? Just to make sure that we are properly guarding (in the tests) against bad values coming through...